maven/bnd-export-maven-plugin/src/main/java/aQute/bnd/maven/export/plugin/ExportMojo.java

```diff
@@ -70,95 +50,46 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 		} catch (Exception e) {
 			throw new MojoExecutionException(e.getMessage(), e);
 		}
+
+		if (errors > 0)
+			throw new MojoExecutionException(errors + " errors found");
 	}
 
-	private void export(File runFile) throws MojoExecutionException, Exception, IOException {
+	private void export(File runFile) throws Exception {
 		if (!runFile.exists()) {
-			throw new MojoExecutionException("Could not find bnd run file " + runFile);
+			logger.error("Could not find bnd run file " + runFile);
+			errors++;
+			return;
 		}
-		try (StandaloneRun run = new StandaloneRun(runFile)) {
+		try (Bndrun run = new StandaloneBndrun(runFile)) {
 			Workspace workspace = run.getWorkspace();
 			workspace.setOffline(session.getSettings().isOffline());
 			for (RepositoryPlugin repo : workspace.getRepositories()) {
 				repo.list(null);
 			}
 			run.check();
 			if (!run.isOk()) {
-				throw new MojoExecutionException("Initializing the workspace failed " + run.getErrors());
+				logger.error("Initializing the workspace failed " + run.getErrors());
```

Comment:
I think we need to warn _always_ and only report error in the `!run.isOk()` case right which already checks for errors but also checks if `-failok` flag and we already decided to return at that point, right?